.circleci/config.yml

```
@@ -115,10 +115,10 @@ jobs:
 
   TPU-tests:
     docker:
-      - image: circleci/python:3.7
+      - image: circleci/python:3.9
     environment:
-      - XLA_VER: 1.9
-      - PYTHON_VER: 3.7
+      - XLA_VER: 1.12
+      - PYTHON_VER: 3.9
```

We could build it on our own, but should we maybe keep testing with Python 3.7? because
1. it seems like they test XLA with 3.7 only https://github.com/pytorch/xla/blob/10db402a64309daede957462c66d5664de182d48/.circleci/config.yml
2. they don't publish wheels for later versions (https://github.com/pytorch/xla/issues/3662)
```suggestion
      - PYTHON_VER: 3.7
```

---

Thanks, let's try it